package.json

```diff
@@ -1,55 +1,38 @@
 {
   "name": "qb-schema",
+  "version": "2.1",
```

Comment:
We should probably bump the version number (per [this comment](https://github.com/quizbowl/schema/pull/80#discussion_r289256718)).